components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -931,7 +931,7 @@ def ice(self, args):
         else:
             self.ctx.call(command)
 
-    @admin_only()
+    @admin_only(AdminPrivilegeReadSession)
```

Comment:
This is a rather strong privilege. What error was being returned -- what powers are actually necessary? (Haven't found the card yet.)

Reply:
Ah -- it's by analogy with @joshmoore's caution with `cleanse`?

Reply:
similar to cleanse

Reply:
Okay, no objections if the caution's warranted.

Reply:
at the moment nobody can run it including root

Reply:
Ahh, I'm afraid I don't recall the permissible ways to write `@admin_only` and never understood the implementation -- @joshmoore, could the bug be in the decorator? Not blocking this PR anyway.

Reply:
I think this could be achieved with `@admin_only(full_admin=True)` *however* that appears to be the default, which doesn't explain why this was failing previously.